kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -1088,6 +1088,30 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
       'Error message that displays if a learning resource cannot be found on the device being used currently.',
   },
 
+  // Device upgrades recommended
+  currentDeviceUsingIE11: {
+    message: 'You seem to be using Internet Explorer 11.',
+    context:
+      'Displayed on a device that is using Internet Explorer 11, as part of a message encouraging the user to upgrade.',
+  },
+  userDevicesUsingIE11: {
+    message: 'Users accessing Kolibri seem to be using Internet Explorer 11.',
+    context:
+      'Displayed to an admin, where devices on their network are using Internet Explorer 11, as part of a message encouraging the user to upgrade.',
+  },
+  browserSupportWillBeDroppedIE11: {
+    message:
+      'Please note that  support for this browser will be dropped in the upcoming version 0.17, so please install another browser to continue working with Kolibri. We recommend Mozilla Firefox or Google Chrome.',
+    context:
+      'Displayed to users of kolibri where one or more devices on the network are using Internet Explorer 11, as part of a message encouraging the user to upgrade.',
+  },
+  pythonSupportWillBeDropped: {
+    message:
+      'Please note that support for Python 2.7 will be dropped in the upcoming version 0.17. Upgrade your Python version to Python 3.7+ to continue working with Kolibri. More recent versions of Python 3 are recommended.',
+    context:
+      'Displayed to users of kolibri where one or more devices on the network are using Internet Explorer 11, as part of a message encouraging the user to upgrade.',
```

Comment:
This string is about Python, but the context still refers to IE11?

Reply:
Good catch Radina!